### PR TITLE
Patch update streaming docs

### PR DESCRIPTION
**Pull request by tkatsoulas**
##### Summary

There are two (overlapping) sections in our docs about Agent streaming components.

1. https://github.com/netdata/netdata/blob/master/streaming/README.md
2. https://github.com/netdata/netdata/tree/master/docs/metrics-storage-management

This PR obsoletes (but not deletes) the `streaming/README.md` doc, every update should be made on  https://github.com/netdata/netdata/tree/master/docs/metrics-storage-management docs which are part of the learn's TOC tree.

##### Test Plan

Part of learn

### CODE REVIEW

File: docs/metrics-storage-management/reference-streaming.mdx [hunk 1]
```diff
@@ -320,6 +332,102 @@ In the following example, the proxy receives metrics from a child node using the
     default memory mode = dbengine
 ```
 
+### Streaming compression
+
+[![Supported version Netdata Agent release](https://img.shields.io/badge/Supported%20Netdata%20Agent-v1.33%2B-brightgreen)](https://github.com/netdata/netdata/releases/latest)
+
+[![Supported version Netdata Agent release](https://img.shields.io/badge/Supported%20Netdata%20stream%20version-v5%2B-blue)](https://github.com/netdata/netdata/releases/latest)
```
[the remark this answers is not in the document]
Comment (tkatsoulas) [reply docs/metrics-storage-management/reference-streaming.mdx:339]: that's a great question: https://github.com/netdata/netdata/blame/67b1936e161e71eea2ecc1960d265c88a2dfd80e/streaming/README.md#L334-L337

@odynik any ideas?

[the remark this answers is not in the document]
Comment (MrZammler) [reply docs/metrics-storage-management/reference-streaming.mdx:339]: I think it refers to the internal stream version that two agents must negotiate between them in order for compression to be working.

In logs we refer to this as `established communication with a parent using protocol version 5 - ready to send metrics...`, i.e. protocol.

[the remark this answers is not in the document]
Comment (MrZammler) [reply docs/metrics-storage-management/reference-streaming.mdx:339]: Agree yes, maybe not needed especially if it's not explained to the user what exactly it means...

While on the subject: Could we should a badge or two here:

image

To indicate the child is connected using e.g. compression and or SSL ?

File: docs/metrics-storage-management/reference-streaming.mdx [hunk 2]
```diff
@@ -320,6 +332,102 @@ In the following example, the proxy receives metrics from a child node using the
...
+netdata INFO  : STREAM_SENDER[child01] : STREAM child01 [send to my.parent.IP]: waiting response from remote netdata...
+netdata INFO  : STREAM_SENDER[child01] : Stream is uncompressed! One of the agents (my.parent.IP <-> child01) does not support compression OR compression is disabled.
+netdata INFO  : STREAM_SENDER[child01] : STREAM child01 [send to my.parent.IP]: established communication with a parent using protocol version 4 - ready to send metrics...
+netdata INFO  : WEB_SERVER[static4] : STREAM child01 [send]: sending metrics...
+```
+
+#### How to enable stream compression
+Netdata Agents are shipped with data compression enabled by default. You can also configure which streams will use compression.
+
+With enabled stream compression, a Netdata Agent can negotiate streaming compression with other Netdata Agents. During the negotiation of streaming compression both Netdata Agents should support and enable compression in order to communicate over a compressed stream. The negotiation will result into an uncompressed stream, if one of the Netdata Agents doesn't support **or** has compression disabled.
```
Comment (DShreve2) [inline docs/metrics-storage-management/reference-streaming.mdx:395]:
```suggestion
With enabled stream compression, a Netdata Agent can negotiate streaming compression with other Netdata Agents. During the negotiation of streaming compression both Netdata Agents should support and enable compression in order to communicate over a compressed stream.  If one of the Netdata Agents doesn't support **or** has compression disabled, the negotiation will result inan uncompressed stream.
```

**Review by DShreve2 — CHANGES_REQUESTED** [review]

File: docs/metrics-storage-management/reference-streaming.mdx [hunk 3]
```diff
@@ -367,7 +367,7 @@ Note: The `stream-compression` status can be `"enabled" | "disabled" | "N/A"`.
 A compressed data packet is determined and decompressed on the fly.
 
 #### Limitations
-This limitation will be withdrawn asap and is work-in-progress.
+This limitation will be withdrawn ASAP and is a work-in-progress.
```
Comment (ilyam8) [inline docs/metrics-storage-management/reference-streaming.mdx:370]: I think that is a famous Brazilian ASAP (cc @thiagoftsm), I think we better remove the line 😄

Comment (DShreve2) [reply docs/metrics-storage-management/reference-streaming.mdx:370]: We can just say "in a future update" or "in a future release"

**Pull request closed without merging**